File: bin/spark/application/accounts-by-state_project/src/main/scala/stubs/AccountsByState.scala
package stubs

import org.apache.spark.sql.SparkSession

object AccountsByState {
  def main(args: Array[String]) {
    if (args.length < 1) {
      System.err.println("Usage: stubs.AccountByState <state-code>")
      System.exit(1)
    }
 
    val stateCode = args(0)

    println("TODO: Solution not yet implemented")
    
  }
}

File: bin/spark/application/python-stubs/accounts-by-state.py
# Python application stub
import sys

from pyspark.sql import SparkSession

if __name__ == "__main__":
  if len(sys.argv) < 2:
    print(sys.stderr, "Usage: accounts-by-state.py <state-code>")
    sys.exit()
    
  stateCode = sys.argv[1]

  print("TODO: Solution not yet implemented")
